src/utils/sort.ts

```diff
@@ -27,16 +26,16 @@ export function sortByDomain(data, property, direction = 'asc', domain) {
 
 export function sortByTime(data, property, direction = 'asc') {
   return data.sort((a, b) => {
-    const aDate = moment(a[property]);
-    const bDate = moment(b[property]);
+    const aDate = a[property].getTime();
+    const bDate = b[property].getTime();
```

Comment:
It would be a bit nicer developer experience if strings were treated as dates, as then you could use data directly from an API-response without having to convert string dates to Date-instances. 

But then again that might perhaps be come with some performance overhead has you need to test strings to see if they contain a valid date, etc. but that is probably negligible..